### ISSUE TITLE
fish ASCII logo usage

### ISSUE THREAD

**Issue by kopischke** [fish-shell/fish-shell#7915]
As I mentioned before, I have been developing a fish syntax extension for Panic’s Nova editor (macOS only, I’m afraid). It has reached a stage where I am feeling like calling it 1.0 and releasing it. Nova has a nice builtin extension library with online fallback, and extensions need an icon to distinguish themselves. Which leads to my question:

Would it be OK with the fish maintainers if I used the fish ASCII art logo in the extension’s icon? The extension is (or will be, as it unpublished yet) open source under the MIT license, so I am not trying to appropriate fish’s logo for commercial gain, and I am happy to include any disclaimer the fish team feels might be necessary to make clear the use of the logo does not constitute an endorsement.

**Comment by faho** [comment]
cc @ridiculousfish 

It would be great if we had an actual, written, license and usage policy for this thing- As far as I am aware we currently do not.

**Comment by kopischke** [comment]
I most certainly do not want to be pushy, but may I leave a gentle ‘ping’?

**Comment by ridiculousfish** [comment]
Apologies for the delay, I'm rather behind on github notifications, I'll take a look.

**Comment by ridiculousfish** [comment]
I certainly don't mind the usage, but I wonder how other projects handle this? Are there any examples we could follow?

**Comment by kopischke** [comment]
NPM has a quite sophisticated [Trademark Policy](https://www.npmjs.com/policies/trademark). In regard to the logo, it states that

> Our npm Logos are very recognizable and deserves special treatment. The npm Logos signify us, or a special relationship with us, and you may use them only with our permission. Since the goal is to avoid confusion about you being us, or your relationship with us, context counts. We will consider requests on a case-by-case basis.

They did give permission for logo usage for an NPM extension in for Nova a while back.

**Comment by kopischke** [comment]
> I certainly don't mind the usage

Would that be permission to forge ahead?

**Comment by ridiculousfish** [comment]
Yeah go ahead. It's clear that fish shell needs a published logo usage document, but no reason to block you on that. Thank you for asking!
